bin/install_swiftpm_dependencies

```diff
@@ -1,30 +1,91 @@
 #!/bin/bash -eu
 
-CUSTOM_PACKAGE_RESOLVED_PATH=${1-}
-BUILD_TYPE=${2-}
+print_usage() {
+  echo "Usage:"
+  echo "  Using the \`Package.resolved\` of an Xcode \`.xworkspace\`:"
+  echo "    $0 --workspace PATH --scheme NAME"
+  echo "  Using the \`Package.resolved\` of an Xcode \`.xcodeproj\`:"
+  echo "    $0 --project PATH"
+  echo "  Using the \`Package.resolved\` at the root, managed by \`swift package\` (and not Xcode)"
```

Comment:
Nitpick. I feel this version "looks better"

```suggestion
  echo "  Using the \`Package.resolved\` at the root, managed by \`swift package\` instead of Xcode:"
```

Notice the `:` at the end. If the reword is not of your liking, we'd still "need" the `:` for consistency with the other lines.

```suggestion
  echo "  Using the \`Package.resolved\` at the root, managed by \`swift package\` (and not Xcode):"
```